tmcprototype/SubarrayNode/SubarrayNode/SubarrayNode.py

```diff
@@ -517,19 +527,31 @@ def Scan(self, argin):
         try:
             print(CONST.STR_SCAN_IP_ARG, argin)
```

Comment:
Remove unnecessary print statement. Also check for such instances throughout the code.

Reply:
Print statements removed.
Commit Id: ac26cff01572307acfb8d209b0bd438358042414